src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CreateClient.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using Xunit;
+
+namespace System.IO.Pipes.Tests
+{
+    /// <summary>
+    /// Tests for the constructors for NamedPipeClientStream
+    /// </summary>
+    public class NamedPipeTest_CreateClient : NamedPipeTestBase
+    {
+        [Fact]
+        public static void NullPipeName_Throws_ArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>("pipeName", () => new NamedPipeClientStream(null));
+            Assert.Throws<ArgumentNullException>("pipeName", () => new NamedPipeClientStream(".", null));
+        }
+
+        [Fact]
+        public static void EmptyStringPipeName_Throws_ArgumentException()
+        {
+            Assert.Throws<ArgumentException>(() => new NamedPipeClientStream(""));
+            Assert.Throws<ArgumentException>(() => new NamedPipeClientStream(".", ""));
```

Comment:
Interestingly enough, there are not parameter names associated with the exceptions for a zero-length string. I confirmed that this behavior is present on Desktop as well with the following test:

```
try {
    var ccli = new NamedPipeClientStream("");
}
catch (ArgumentException e)
{
    Console.WriteLine(e.ParamName.Length); // throws ArgumentNullException - e.ParamName is null
}
```